.github/workflows/update-homebrew.yml

```diff
@@ -0,0 +1,34 @@
+name: Update Homebrew
+on: 
+  release:
+    types: [created]
+
+jobs:
+  update-core:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get token
+        id: get-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ vars.PENNY_APP_ID }}
+          private-key: ${{ secrets.PENNY_APP_PRIVATE_KEY }}
+          owner: vapor
+
+      - name: Add path
+        run: echo /home/linuxbrew/.linuxbrew/bin >>"${GITHUB_PATH}"
+        
+      - name: Update Homebrew formula
+        env:
+          HOMEBREW_GITHUB_API_TOKEN: ${{ steps.get-token.outputs.token }}
+        run: |
+          git config --global user.name 'Penny[bot]'
+          git config --global user.email '360798+penny[bot]@vapor.codes'
```

Comment:
Email looks a bit suspect? Or is that was we have set for the app?

Reply:
It's what we've been using; whether it's _correct_ I'm less sure.

Reply:
Turns out this is surprisingly intricate. References:

- https://api.github.com/users/penny-for-vapor%5Bbot%5D
- https://github.com/orgs/community/discussions/24664#discussioncomment-3880274
- https://github.com/Nautilus-Cyberneering/pygithub/blob/main/docs/how_to_sign_automatic_commits_in_github_actions.md

Note (as pointed out in one of those links) that the "user ID" for the app (now saved as the organization variable `PENNY_APP_USER_ID`) is _NOT_ the same as the app's "app ID" (already existed as `PENNY_APP_ID`).